website/addons/gdrive/templates/gdrive_node_settings.mako

```diff
@@ -0,0 +1,84 @@
+
+<div id="driveAddonScope" class="scripted">
+<h4 class="addon-title">
+    Google Drive
+    <small class="authorized-by">
+        <span data-bind="if:nodeHasAuth">
+            authorized by <a data-bind="attr.href: owner">
+            {{ownerName}}
+            </a>
+            % if not is_registration:
+                <a data-bind="click: deauthorize"
+                    class="text-danger pull-right addon-auth">Deauthorize</a>
+            % endif
+        </span>
+
+         <!-- Import Access Token Button -->
+        <span data-bind="if: showImport">
+            <a data-bind="click: importAuth" href="#" class="text-primary pull-right addon-auth">
+                Import Access Token
+            </a>
+        </span>
+
+        <!-- Oauth Start Button -->
+        <span data-bind="if: showTokenCreateButton">
+            <a data-bind="click: createAuth" class="text-primary pull-right addon-auth">
+                Create Access Token
+            </a>
+        </span>
+    </small>
+    </h4>
+
+    <div id="currentFolder" data-bind="if:showFolders()">
+       <p>
+
+        <strong> Current folder:</strong>
+        <a data-bind="attr.href: urls().files">
+            <span data-bind = "text:currentFolder"> None </span>
+        </a>
+       </p>
+
+        <div class="btn-group" >
+        <button data-bind="click:changeFolder" class="btn btn-sm btn-dropbox"> Change Folder</button>
```

Comment:
I believe @samchrisinger has separated much of the shared addon-related css (the rules for `btn-dropbox`, etc.) to a separate file and renamed the class names to be more generic. Once his changes are merged, these class names should be changed.